honeycombio/honeycomb-opentelemetry-swift
# feat(swift): Add a .swift-format config and run swift-format.

## Which problem is this PR solving?

Adds `swift-format` setup to make sure the codebase maintains consistent code formatting style.

## Short description of the changes

Adds:
* a `.swift-format` configuration for standard formatting
* `Makefile` targets for formatting and linting with `swift-format`
* a circleci workflow that should hopefully run `make lint`
* Instructions for how to set up `swift-format`, if needed

I also ran `make format` and `make lint` to make the code pass the new lint checks.

## How to verify that this has the expected result

Edit some code to have non-standard format and then run:
`make lint`
`make format`

Observe that it was changed to be formatted correctly.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208116694462795
